routes/index.js

```diff
@@ -47,8 +47,19 @@ router.post('/user/login', function(req, res) {
         if (err) {
             res.json("Your username is not exist");
         }else if(pass == docs.password){
-          var access_token=docs._id;
-            res.json({'access_token':docs._id});  
+          var now=moment().format("YYYY-MM-DD'T'HH:mm:ss:SSSZ");
+            var loginRecord = new req.login({
```

Comment:
sir if i dont use format here then the time is saved with spaces and creates problem when i am getting it on validation page